src/components/StyledButtons/CloseTabBtn.jsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import CloseTabBtnImg from "../../assets/image/CloseTabBtnBackground.svg";
+
+const BtnStyle = styled.button`
+  background: url(${CloseTabBtnImg});
+  width: 24px;
+  height: 24px;
+  flex-shrink: 0;
+  border: none;
+  position: absolute;
+  top: 16px;
+  right: 16px;
+  z-index: 5;
+`;
+
+function CloseTabBtn({ onClick }) {
+  return <BtnStyle onClick={() => onClick(false)}></BtnStyle>;
```

Comment:
svg 파일을 리액트 컴포넌트로 만들고, 해당 컴포넌트는 div로 wrap해주는 방식은 어떨까요?
그러면 BtnStyle에서 background 속성으로 svg를 불러올 필요가 없어질것같아요!

```suggestion
  return <div onClick={() => onClick(false)}><CloseIcon /></div>;
```